controllers/istiocni/istiocni_controller.go

```diff
@@ -141,32 +142,37 @@ func (r *Reconciler) installHelmChart(ctx context.Context, cni *v1.IstioCNI) err
 		BlockOwnerDeletion: ptr.Of(true),
 	}
 
+	versionName, err := istioversions.ResolveVersionName(cni.Spec.Version)
```

Comment:
Shouldn't this be called `versionNumber` or simply `version`, since it contains the resolved version number, not the name?

Reply:
It is like "v1.24.1", which is a version number prefixed with "v".

Reply:
But that's still a version number, isn't it? Whereas "master" is not a number, but is a version "name" (or alias or whatever).